WordPress/Classes/ViewRelated/Gutenberg/GutenbergMediaPickerHelper.swift

```diff
@@ -31,7 +31,14 @@ class GutenbergMediaPickerHelper: NSObject {
     ///
     fileprivate lazy var devicePhotoLibraryDataSource = WPPHAssetDataSource()
 
-    fileprivate lazy var mediaPickerOptions: WPMediaPickerOptions = {
+    var didPickMediaCallback: GutenbergMediaPickerHelperCallback?
+
+    init(context: UIViewController, post: AbstractPost) {
+        self.context = context
+        self.post = post
+    }
+
+    fileprivate func defaultMediaPickerOptions() -> WPMediaPickerOptions {
```

Comment:
I changed this to a function so that we'd get a fresh `WPMediaPickerOptions` each time rather than mutating a property.